tests/wicked/before_test.pm

```diff
@@ -35,7 +37,8 @@ sub run {
     #preparing directories for holding config files
     assert_script_run('mkdir -p /data/{static_address,dynamic_address}');
     #download script for check interface status
-    $self->get_from_data('wicked/check_interfaces.sh', '/data/check_interfaces.sh', executable => 1) if check_var('WICKED', 'basic');
+    $self->get_from_data('wicked/check_interfaces.sh', '/data/check_interfaces.sh', executable => 1)
```

Comment:
this line already passes Travis check before. So no need to change it. Also in this form, it is confusing. So if you want to change this do it :
```
if check_var('WICKED', 'basic') {
   $self->get_from_data('wicked/check_interfaces.sh', '/data/check_interfaces.sh', executable => 1);
}
```

Reply:
Sorry, it was auto tidy :)